app/furniture/marketplace/split_job.rb

```diff
@@ -0,0 +1,22 @@
+class Marketplace
+  class SplitJob < ApplicationJob
+    attr_accessor :order
+    delegate :vendors_share, :marketplace, to: :order
+    delegate :stripe_api_key, :vendor_stripe_account, to: :marketplace
+
+    sidekiq_options retry: 5
+    sidekiq_retry_in { |count, exception, jobhash| 1.day }
+
+    def perform(order:)
+      self.order = order
+      order.events.create(description: "Payment Split Attempted")
+      Stripe::Transfer.create({
+        amount: vendors_share.to_i,
+        currency: "usd",
+        destination: vendor_stripe_account,
+        transfer_group: order.id
+      }, {api_key: stripe_api_key})
+      order.events.create(description: "Payment Split Completed")
```

Comment:
I think we might want to have a flag on the order `split_completed` that we set to `true`, and before attempting the transfer we check that flag. It makes it easier to avoid double-transfers, and also makes it possible to find Orders with / without the split, for debugging.

Reply:
Are there any other conditions we should check here before trying to transfer the split? For example, check that the order status is still `paid`.

Reply:
I am a bit hesitant to add checking for `paid?` since we could add `delivered` or `ready_for_pickup` or `in_transit` all of which would no longer count as `paid?`...

But I do think we should have a flag of some kind. I went with an id-and-also-flag since keeping the context of which transfer did the split doesn't seem like a bad idea?